src/matchers/mock/toBeRequestedTimes.ts

```diff
@@ -4,7 +4,7 @@ import { waitUntil, enhanceError, compareNumbers } from '../../utils'
 import { runExpect } from '../../util/expectAdapter'
 import { numberError } from '../../util/formatMessage'
 
-export function toBeRequestedTimesFn(received: Mock, expected: number| ExpectWebdriverIO.NumberOptions = {}, options: ExpectWebdriverIO.StringOptions = {}): any {
+export function toBeRequestedTimesFn(received: Mock & {browser: WebdriverIO.Browser}, expected: number| ExpectWebdriverIO.NumberOptions = {}, options: ExpectWebdriverIO.StringOptions = {}): any {
```

Comment:
In mock functions I can't get browser instance. But I see that `Interception` type from which `Mock` is inherited has `browser` field - https://github.com/webdriverio/webdriverio/blob/428a9d729ae6231968a60908732fa3f607d195e9/packages/webdriverio/src/utils/interception/index.ts#L30

So I add this small hack which is working. If you have any ideas how to make it better, then I'm ready to fix it.